src/store.js

```diff
@@ -0,0 +1,17 @@
+import { combineReducers, createStore } from "redux";
```

Comment:
Could comment some different way, please.

Reply:
we have a part of code that deals with the application state - redux store.
So I think it would be natural to keep them in a folder called store. It can contain the main file - store, and different slices.
But instead you have a single file - store.ts in root directory and slice files in directory called features

You can check the other developers' codes for naming conventions

Reply:
I fixed it the way you said, I created a folder store and placed in it the store.ts and the folder features. If you remember in my last update I was thinking of creating folder named redux and place in it those part of code, but I believe your option is better.